nuv/wsk.go

```diff
@@ -57,6 +57,16 @@ func Wsk(args ...string) error {
 	return commands.Execute()
 }
 
+func setWskConfigFile() error {
+	path, err := getWskPropsFilePath()
+	if err != nil {
+		fmt.Println("Did you create a cluster? Try nuv setup --devcluster")
+		return err
```

Comment:
Hey @Deviad this can be simplified as 
return fmt.Errorf("Did you create a cluster? Try nuv setup --devcluster")
it is idiomatic Go